feat(desktop-modeler): hint on desktop modeler <> IDE interplay (#6037)
camunda/camunda-docs: docs/components/modeler/desktop-modeler/index.md

id: index
title: Desktop Modeler
sidebar_label: About

> description: "Camunda Desktop Modeler is a desktop app for modeling BPMN, DMN, and Forms, compatible with Camunda 7 and Camunda 8. As a Camunda implementation tool it works alongside your favorite IDE, in the professional software development environment of your choice"

Reviewer: ```suggestion
description: "Camunda Desktop Modeler is a companion desktop application for modeling BPMN, DMN, and Forms, compatible with Camunda 7 and Camunda 8. Designed to integrate seamlessly into professional software development workflows, it is intended to be used alongside your preferred IDE—not as a standalone tool."
```

WDYT of this?

> description: "Camunda Desktop Modeler is a companion desktop application for modeling BPMN, DMN, and Forms, compatible with Camunda 7 and Camunda 8. Designed to integrate seamlessly into professional software development workflows, it is intended to be used alongside your preferred IDE—not as a standalone tool."

Author: ```suggestion
description: "Camunda Desktop Modeler is a desktop application for modeling BPMN, DMN, and Forms, compatible with Camunda 7 and Camunda 8. As a companion tool to your favorite IDE it is designed to integrate seamlessly into your professional software development environment."
```

I'd like to prevent the framing of "Don't us it standalone", but instead highlight the "better together (with an IDE, for implementing Camunda)" story.

What do you think @christinaausley?

Author: I'd also consider to align both this description and the introduction of the page (README).

---

<span class="badge badge--cloud">Camunda 7 and 8</span>

> Desktop Modeler is a desktop application for modeling BPMN, DMN, and Forms. As a Camunda implementation tool it supports you in building executable diagrams—alongside your favorite IDE, in the professional software development environment of your choice.

Reviewer: ```suggestion
Desktop Modeler is a desktop application for modeling BPMN, DMN, and Forms. As part of the Camunda implementation toolkit, it helps you create executable diagrams while working alongside your preferred IDE, integrated into your professional software development environment.
```

Author: Great one! :star:
